### PR TITLE
Bug 573296 - [Passage] rework EMF migration facilities, part 6

### PR DESCRIPTION
**Pull request by ruspl-afed**
Rework migration for features: use resource handler instead of external
OSGi component

Signed-off-by: Alexander Fedorov <alexander.fedorov@arsysop.ru>

### CODE REVIEW

File: bundles/org.eclipse.passage.lic.features.model/plugin.xml [hunk 1]
```diff
@@ -22,4 +22,11 @@
             genModel="model/features.genmodel"/>
    </extension>
 
+   <extension point="org.eclipse.emf.ecore.extension_parser">
+      <!-- @generated features -->
+      <parser
+            type="features"
```
Comment (eparovyshnaya) [inline bundles/org.eclipse.passage.lic.features.model/plugin.xml:28]: should it correspond to `.features_xmi` file extension?

File: bundles/org.eclipse.passage.lic.features.model/src-gen/org/eclipse/passage/lic/features/model/util/FeaturesResourceImpl.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	public FeaturesResourceImpl(URI uri) {
+		super(uri);
+	}
+
+	@Override
+	protected void init() {
+		super.init();
+		Map<Object, Object> options = getDefaultLoadOptions();
+		options.put(OPTION_RECORD_UNKNOWN_FEATURE, Boolean.TRUE);
+		options.put(OPTION_RESOURCE_HANDLER, new FeaturesResourceHandler());
```
Comment (eparovyshnaya) [inline bundles/org.eclipse.passage.lic.features.model/src-gen/org/eclipse/passage/lic/features/model/util/FeaturesResourceImpl.java:46]: The coolest thing for the latest time!

**Review by eparovyshnaya — APPROVED** [review]
Awesome!11

File: bundles/org.eclipse.passage.lic.features.model/src-gen/org/eclipse/passage/lic/features/model/util/FeaturesResourceImpl.java (hunk 2 above)
Comment (ruspl-afed) [reply bundles/org.eclipse.passage.lic.features.model/src-gen/org/eclipse/passage/lic/features/model/util/FeaturesResourceImpl.java:46]: 10x

File: bundles/org.eclipse.passage.lic.features.model/plugin.xml (hunk 1 above)
Comment (ruspl-afed) [reply bundles/org.eclipse.passage.lic.features.model/plugin.xml:28]: Good point!

**Pull request merged**